pkg/progress/progress.go

```diff
@@ -110,7 +126,7 @@ func (m *Manager) Status(progress string) (process, leftSeconds, currentSpeed fl
 	defer m.RUnlock()
 	if p, exist := m.progesses[progress]; exist {
 		process = 1 - p.left/p.total
-		speedPerSec := (p.total - p.left) / time.Since(p.startTime).Seconds()
+		speedPerSec := (p.last - p.left) / time.Since(p.lastTime).Seconds()
```

Comment:
I think this approach is unreliable and has a high interpretation cost. Because `Status()` will return zero speed and infinite remaining time after an `UpdateProgressLeft()` operation whose `last` is equal to `left` or an initialization. It would be unwise to expose this inaccurate result to the user.